src/components/forms/components/SearchAsYouTypeAjax.js

```diff
@@ -0,0 +1,699 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { _ } from 'underscore';
+import memoize from 'memoize-one';
+import ReactTooltip from 'react-tooltip';
+
+import { Fade } from './../../ui/Fade';
+
+import { ajax, object } from './../../util/';
+import { valueTransforms } from './../../util';
+
+import { Alerts } from './../../ui/Alerts';
+
+import { LinkToSelector } from './LinkToSelector';
+import { SearchSelectionMenu } from './SearchSelectionMenu';
+
+export class SearchAsYouTypeAjax extends React.PureComponent {
+
+    static getRegexQuery(value, filterMethod) {
```

Comment:
We should remove this method as well as `filterOptions`, at least from this component, since only relevant for SearchAsYouTypeLocal.

Reply:
Really good catch; didn't even realize it wasn't being used anywhere.